jquery.ui.rotatable.js

```diff
@@ -107,10 +107,25 @@ $.widget("ui.rotatable", $.ui.mouse, {
           '-webkit-transform-origin',
           this.options.rotationCenterX + '% ' + this.options.rotationCenterY + '%'); /* Chrome, Safari, Opera */
 
-        this.element.css('transform','rotate(' + angle + 'rad)');
-        this.element.css('-moz-transform','rotate(' + angle + 'rad)');
-        this.element.css('-webkit-transform','rotate(' + angle + 'rad)');
-        this.element.css('-o-transform','rotate(' + angle + 'rad)');
+        var transforms = 'rotate(' + angle + 'rad)';
```

Comment:
So, if I understand this change correctly, there would be a new option called 'transforms' which gets applied to the CSS of the rotatable element?

If that's the case, can you also include a change to the documentation to include your new feature? Thanks!